types.go

```diff
@@ -1050,8 +1050,7 @@ type TKEY struct {
 
 // TKEY has no official presentation format, but this will suffice.
 func (rr *TKEY) String() string {
-	s := "\n;; TKEY PSEUDOSECTION:\n"
-	s += rr.Hdr.String() + " " + rr.Algorithm + " " +
+	s := ";" + rr.Hdr.String() + " " + rr.Algorithm + " " +
 		strconv.Itoa(int(rr.KeySize)) + " " + rr.Key + " " +
 		strconv.Itoa(int(rr.OtherLen)) + " " + rr.OtherData
```

Comment:
This seems to be missing about half of the [RDATA fields](https://tools.ietf.org/html/rfc2930#section-2)... my advice would be to adopt the [approach from BIND](https://gitlab.isc.org/isc-projects/bind9/blob/master/lib/dns/rdata/generic/tkey_249.c#L116) as a de facto standard:
```
$algorithm.as.domain.name. $inception_as_decimal $expiration_as_decimal $mode $error $key_size (
	$key_data_as_base64 ) $other_size ( $other_data_as_base64 )
```